railties/lib/rails/test_unit/reporter.rb

```diff
@@ -15,8 +15,12 @@ def aggregated_results # :nodoc:
       filtered_results.reject!(&:skipped?) unless options[:verbose]
       filtered_results.map do |result|
         location, line = result.method(result.name).source_location
-        "bin/rails test #{location}:#{line}"
+        "bin/rails test #{relative_path_for(location)}:#{line}"
       end.join "\n"
     end
+
+    def relative_path_for(file)
+      file.sub(/^#{Dir.pwd}\/?/, '')
```

Comment:
I experimented with it, but I couldn't get it to work like I wanted it to. And it was useless on absolute paths (which is what we're dealing with here).